### DISCUSSION TITLE
Add ability to remove your own picked maps from multi queue

### DISCUSSION THREAD

**Discussion by respektive** [ppy/osu#15831] [Ideas] [answered]
There should be a way to remove maps again if you decide, you actually don't wanna play it. A way to vote skip a map would also be nice.

**Comment by bdach** [comment, marked as answer]
known missing feature. will be coming in upcoming releases.

**Reply by peppy** [reply]
Will be resolved via https://github.com/ppy/osu/pull/16010.
